include/zephyr/bluetooth/l2cap.h

```diff
@@ -349,6 +349,9 @@ struct bt_l2cap_chan_ops {
 	 *  @kconfig{CONFIG_BT_L2CAP_SEG_RECV} is enabled and seg_recv is
 	 *  supplied.
 	 *
+	 *  If the application returns -EINPROGRESS, the reference
+	 *  @p buf was moved.
```

Comment:
What does `moved` mean in this context?

Reply:
@alwa-nordic that means the note is not clear enough. 
@Thalley it means the reference was moved into the application. The app now owns `buf`.